ui-library-filesharing-ui-components/app/src/ChatComponents.tsx

```diff
@@ -46,7 +46,7 @@ export default function ChatComponents(): JSX.Element {
         },
       })
       .then((res) => {
-        completeFileUpload(file.name, { name: file.name, extension, url: res.data.url });
+        completeFileUpload(file.name, { attachmentType: 'fileSharing', id: v4(),name: file.name, extension, url: res.data.url });
```

Comment:
Should we use `uniqueFileName` as `id` value here or use the `id` value as part of the path of `url` in 

```
axios
      .request({
        method: "post",
        url: `/api/UploadFileToAzureBlobStore?filename=${uniqueFileName}`,
        data: data,
        onUploadProgress: (p) => {
          updateFileUploadProgress(file.name, p.loaded / p.total);
         },
       })
```